### DISCUSSION TITLE
renderHeadToString not available at '@unhead/vue';

### DISCUSSION THREAD

**Discussion by victorlmneves** [vueuse/head#190] [Q&A] [answered]
Hi

I was using @vueuse/head and now when I was updating to this version I saw an error when replacing my imports to @unhead/vue where I was importing renderHeadToString

I tried to find something about this but didn't find it. In the readme it only mentions the replacement of the library name :/

Thanks

**Comment by harlan-zw** [comment]
Hi, please see the [Unhead SSR](https://unhead.unjs.io/setup/ssr/installation) guide. Will look to improve the migration docs.

**Comment by victorlmneves** [comment]
> Hi, please see the [Unhead SSR](https://unhead.unjs.io/setup/ssr/installation) guide. Will look to improve the migration docs.

So if I understood correctly the feature was moved to a dedicated package.
Instead of
```
import { renderHeadToString } from '@vueuse/head';
...
const { headTags, htmlAttrs, bodyAttrs } = await renderHeadToString(head);
```


It needs to be
```
import { renderSSRHead } from '@unhead/ssr'
...
const { headTags, htmlAttrs, bodyAttrs } = await renderSSRHead(head);
```
Right @harlan-zw ?

Thanks

**Reply by harlan-zw** [reply, marked as answer]
Correct, this was done to improve tree-shaking
